doc/source/serve/develop-and-deploy.md

```diff
@@ -1,6 +1,6 @@
 (serve-develop-and-deploy)=
 
-# Develop and deploy an ML application
+# Develop and Deploy an ML application
```

Comment:
why capitalize this one instead of use sentence case? Is it because it's the top-level heading?

if we capitalize "Deploy," we should also capitalize "Application"

Reply:
Yes, and good call. I will capitalize Application.

Reply:
Ah! Good catch. Will fix.

Reply:
Eventually, we should move to sentence case even for the top-level heading, but until we can do them all at once, consistency trumps.